common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -497,4 +497,15 @@ trait FeatureSwitches {
     exposeClientSide = true
 
   )
+
+  val ArticleWithStructuredRecipe = Switch(
+    SwitchGroup.Feature,
+    "is-article-with-structured-recipe-data",
+    "changes design of articles with strucutred recipe data",
+    owners = Seq(Owner.withGithub("tsop14"), Owner.withGithub("blongden73")),
+    safeState = Off,
+    sellByDate = new LocalDate(2017, 12, 1),
```

Comment:
That's pretty far away. When do you think the feature would be actually ready and tested?